apps/actors/lib/actors/registry/actor_registry.ex

```diff
@@ -247,25 +247,29 @@ defmodule Actors.Registry.ActorRegistry do
       |> Enum.uniq()
       |> List.first()
 
-    actual_opts = actual_host_member.host_function.opts
-    actual_actors = actual_host_member.host_function.actors
+    if actual_host_member do
```

Comment:
This is meant to be defensive only

In some cases, due to eventual inconsistency, it will throw a actual_host_member null pointer error due to not being synqued yet

Reply:
It could be, but if it does, it would be okay to let the exception propagate anyway. Right?

Reply:
I don't think so, due to being a GenServer